src/styles/blocks/card.scss

```
@@ -0,0 +1,83 @@
+.card {
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    display: flex;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+    align-items: center;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-black;
+  }
+
+  &__id {
+    margin-bottom: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-grey;
+  }
+
+  &__mark {
+    display: flex;
+    margin-bottom: 24px;
+    justify-content: space-between;
+    align-items: center;
```

Можна винести в плейсхолдер.
```suggestion
   display: flex;
    justify-content: space-between;
    align-items: center;
```